src/main/java/Branch.java

```diff
@@ -0,0 +1,48 @@
+public class Branch {
+
+    private int ID;
+    private String Name;
+    private String Address;
+    private String Owner;
+    private Exam exams;
```

Comment:
camel case
exams -> Exams

In addition - if you are willing to keep one exam for branch - call it "exam" if you willing to keep few exams for branch use some container as - "List" or "Array" or whatever..